### PR TITLE
Exercise scenarios sortable

### PR DESCRIPTION
**Pull request by MariaSoleim**
This pull request will make it possible to sort on all different columns in exercise scenarios. There are more tables to sort, but you can have a look at how I've done this before I implement anything on the other tables.

capture

### CODE REVIEW

**Comment by MariaSoleim** [comment]
I will look into it :)

File: src/components/exercise-scenarios/ExerciseScenarios.js [hunk 1]
```diff
@@ -1,96 +1,125 @@
...
+                </Tooltip>
+              </TableCell>
             );
-          })}
-        </TableBody>
-      </Table>
-    </Paper>
-  </AppPage>
-);
+          }, this)}
```
Comment (JamiesWhiteShirt) [inline src/components/exercise-scenarios/ExerciseScenarios.js]: When using arrow function supplying a `this` context as done with `.map(..., this)` here is unnecessary. The arrow function captures the `this` context, so `, this` can be safely removed.

File: src/helpers/table-sort-helper.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  return stabilizedThis.map(el => el[0]);
+}
+
+function getSorting(order, orderBy) {
+  return order === "desc"
+    ? (a, b) => desc(a, b, orderBy)
+    : (a, b) => -desc(a, b, orderBy);
+}
+
+export { stableSort, getSorting }
```
Comment (JamiesWhiteShirt) [inline src/helpers/table-sort-helper.js]: These functions can also be exported inline if desired, e.g. `export function stableSort() { ... }`.

Nothing that has to be fixed, just worth knowing.

File: src/components/exercise-scenarios/ExerciseScenarios.js [hunk 3]
```diff
@@ -1,96 +1,125 @@
...
+}
+
+export default class ExerciseScenarios extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      order: "asc",
+      orderBy: "week",
+      data: scenarios,
```
[the remark this answers is not in the document]
Comment (MariaSoleim) [reply src/components/exercise-scenarios/ExerciseScenarios.js]: Yes. It's necessary to know which parameter the list is sorted on and if it is ascending or descending to know if a column is clicked a second time and what the new order should be.

**Pull request merged**